### DISCUSSION TITLE
Nouvelles fonctionnalités pour les familiers

### DISCUSSION THREAD

**Discussion by BastLast** [Crownicles/Crownicles#689] [Suggestions fermées]
Un set de nouvelles fonctionnalités pour les pets :

- [ ]  les pets ont une stats de santé en plus de la stat d'amour
- [ ] ajout d'un nouveau menu permettant d'interagir avec le pet (caresser, soigner, jouer) 
- [ ] jouer avec un familier occupe le joueur pendant 15 minutes (donne +2 d'amour et +2 en santé) 
- [ ] le cooldown pour jouer ou caresser un pet est de 20 minutes * la rareté du pet (divisé par 2 pour la caresse)
- [ ] caresser un familier occupe le joueur pour une durée random entre 0 et 5 minutes et donne 1 d'amour et 1 de santé)
- [ ] soigner nécessite des médicaments (nouvelle nourriture pour pet) (donne 0 d'amour, mais +10 en santé)
- [ ] si la santé d'un familier tombe à 0 il meurt, mais si elle est au max il gagne +1 d'amour à chaque fois qu'il gagne de l'amour
- [ ] Ajout d'un mini event faisant perdre de la santé aux pets,
- [ ] Ajout d'une nouvelle classe ou les familiers du joueur prennent 50 % des dégâts qui lui sont infligés à sa place, les stats de cette classe seront moyenne pour le reste (en gros classe intéressante surtout earlygame car permet de tanker des pv dans un pet qui a beaucoup de pv) (peut etre ca serait une classe tier 1 2 et 3 et pas dispo au tier 4 ?
- [ ] Ajout d'un mini event ou le joueur a une récompense si son pet est en bonne santé
- [ ] chaque pet a une santé max qui change en fonction du type de pet (un ours a plus de pv qu'un pigeon)
- [ ] Léger buff de la nourriture (légère réduction du cout ou des gains d'amour)

**Comment by Pietagorh** [comment]
Globalement j'suis vraiment contre, pour les familiers faut pas que ce soit trop complexe, faudrait juste les ajouter dans les fight et quelques petits trucs (#573)
>  Ajout d'une nouvelle classe ou les familiers du joueur prennent 50 % des dégâts qui lui sont infligés à sa place, les stats de cette classe seront moyenne pour le reste (en gros classe intéressante surtout earlygame car permet de tanker des pv dans un pet qui a beaucoup de pv) (peut etre ca serait une classe tier 1 2 et 3 et pas dispo au tier 4 ?

Par rapport au fait que ce soit une classe early game, j'suis contre, vu que t'as pas de bons pets quand tu débutes, et que ce serait surtout utile pour les gens qui ont des très bons pets
> Ajout d'un mini event faisant perdre de la santé aux pets

Et pas pour lui en faire gagner ?
> chaque pet a une santé max qui change en fonction du type de pet (un ours a plus de pv qu'un pigeon)

Mais du coup faudrait refaire les classements, parce qu'il y a des 2 :star: qui sont moins résistants que des 1 :star: irl, ou alors donner plusieurs utilités à chaque pet (toujours #573)

**Comment by BastLast** [comment]
l'idée est à la poubelle :)

**Comment by Chevalier2** [comment]
Cela pourrait être intéressant. Seulement si les pets meurent vraiment avec difficulté. Surtout les bons pets. Qu'il n'y ait pas besoin de payer pour les garder en vie, ni besoin de perdre trop de temps. Cela créerait plus d'interactions avec les pets. Par contre, l'achat de nourriture serait délaissé, puisqu'il serait bien plus simple de les dresser. Peut-être trop même. Jouer, +2 points d'affection, caresser +1... Cela fait beaucoup.
